lang/csharp/src/apache/test/Schema/SchemaTests.cs

```diff
@@ -548,12 +549,79 @@ public void TestLogicalPrimitive(string s, string baseType, string logicalType)
             testToString(sc);
         }
 
+        // Make sure unknown type is carried thru to LogicalTypeName
         [TestCase("{\"type\": \"int\", \"logicalType\": \"unknown\"}", "unknown")]
         public void TestUnknownLogical(string s, string unknownType)
         {
-            var err = Assert.Throws<AvroTypeException>(() => Schema.Parse(s));
+            //var err = Assert.Throws<AvroTypeException>(() => Schema.Parse(s));
+            //Assert.AreEqual("Logical type '" + unknownType + "' is not supported.", err.Message);
```

Comment:
Are these lines needed ?

Reply:
they are not, those were the original lines that i replaced